### PR TITLE
Support patient access for r4 Slot read and search capability

### PR DESCRIPTION
**Pull request by Sanvenky**
Description
----

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Comment by Sanvenky** [comment]
Screenshot(s) of changes attached before changes merged:

image

**Review by aki-mathur — APPROVED** [review]

**Review by NeethuKP — APPROVED** [review]

File: lib/resources/example_json/r4_examples_slot.rb [hunk 1]
```diff
@@ -243,5 +333,77 @@ module Resources
...
+                '=Slot%3Aschedule&-pageContext=2d46d307-9a4c-4dc7-a1c2-88f1282d5aec&-pageDirection=NEXT'
+        }
+      ],
+      'entry': [
+        {
+          'fullUrl': 'https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Slot/4047611-32216049-61518614-0',
+          'resource': R4_SLOT_PATIENT_ACCESS_ENTRY
+        },
+        {
+          'fullUrl': 'https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Schedule/4047611-25442717-61519334-45',
```
Comment (ashu-bdvt) [inline lib/resources/example_json/r4_examples_slot.rb:361]: nit: Shouldn't Schedule ID be the same as slot?

**Review by ashu-bdvt — APPROVED** [review]

File: lib/resources/example_json/r4_examples_slot.rb (hunk 1 above)
Comment (Sanvenky) [reply lib/resources/example_json/r4_examples_slot.rb]: addressed

**Comment by sydneanschutz** [comment]
Validated on live:
Screen Shot 2021-02-18 at 1 26 26 PM

**Pull request merged**